GitCommands/Settings/SettingsCache.cs

```diff
@@ -16,19 +16,14 @@ public SettingsCache()
 
         public void Dispose()
         {
-            DisposeImpl();
+            Dispose(true);
             GC.SuppressFinalize(this);
         }
 
-        protected virtual void DisposeImpl()
+        protected virtual void Dispose(bool disposing)
```

Comment:
What is the purpose of the `disposing` argument? It is always being invoked with the `true` value.